projects/cashmere-examples/src/lib/modal-overview/modal-overview-example-modal.component.html

```diff
@@ -1,12 +1,26 @@
 <hc-modal>
     <hc-modal-header>Modal Header Title</hc-modal-header>
-    <hc-banner type="warning">
+    <hc-banner type="warning" *ngIf="activeModal.data.tight">
         <span hcBannerStamp>WARNING</span>
         You are about to export data which may include Protected Health Information (PHI)
     </hc-banner>
-    <hc-modal-body>Data: {{ activeModal.data }}</hc-modal-body>
+    <hc-modal-body>Data: {{ activeModal.data.content }}</hc-modal-body>
     <hc-modal-footer>
-        <button hc-button buttonStyle="link" (click)="cancel()">Cancel</button>
-        <button hc-button buttonStyle="primary" (click)="close()">OK</button>
+        <button
+            hc-button
+            buttonStyle="link"
+            (click)="cancel()"
+            [size]="activeModal.data.tight ? 'md' : 'sm'"
```

Comment:
This looks like it's the opposite of what I'd expect. I might just be missing something though... I'd love to chat with you about this in more detail to make sure I understand what I'm seeing.